fe/fe-core/src/main/java/org/apache/doris/qe/StmtExecutor.java

```diff
@@ -582,31 +592,55 @@ private void handleSetStmt() {
     }
 
     // Process a select statement.
-    private void handleQueryStmt() throws Exception {
+    private boolean handleQueryStmt() throws Exception {
         // Every time set no send flag and clean all data in buffer
         context.getMysqlChannel().reset();
         QueryStmt queryStmt = (QueryStmt) parsedStmt;
+        // Use connection ID as session identifier
+        NamedKey namedKey = new NamedKey(String.valueOf(context.getConnectionId()),
```

Comment:
We had a `originStmt.originStmt`, which is the origin sql string. how about use it instead of `queryStmt.toSql()`?